### ISSUE TITLE
Idea: exporting wrapped components

### ISSUE THREAD

**Issue by oreqizer** [erikras/redux-form-material-ui#1]
The flow would then become:

```javascript
import { WrappedTextField } from 'redux-form-material-ui';

const MyForm = props =>
  <form onSubmit={props.handleSubmit}>
    <Field
      name="username"
      component={WrappedTextField}
      floatingLabelText="Username"
    />
  </form>;

export default reduxForm({
  form: 'myForm',
})(MyForm);
```

This allows for a better tree shaking with webpack2, is more explicit and less error prone in my opinion (no string component matching).

**Comment by erikras** [comment]
Those are good arguments against the "adapter" API, which was somewhat conceived in a late night coding binge. How about naming?

`WrappedTextField`

`ReduxFormTextField` / `RfTextField`

or just...

```js
import { RadioButtonGroup, TextField } from 'redux-form-material-ui';
import  { RadioButton } from 'material-ui';
```

**Comment by oreqizer** [comment]
I would simply name it the way `material-ui` has it. Less overhead to keep in mind, and I don't think it would confuse anyone if they keep their form and field usage consistent (meaning they stick to `redux-form` and `material-ui`).

**Comment by erikras** [comment]
To be clear, you're voting for Option 3?

```js
import { RadioButtonGroup, TextField } from 'redux-form-material-ui';
import  { RadioButton } from 'material-ui';
```

**Comment by oreqizer** [comment]
yep

**Comment by erikras** [comment]
Great suggestion. Released as [`v2.0.0`](https://github.com/erikras/redux-form-material-ui/releases/tag/v2.0.0).
